### PR TITLE
Switch to Github Actions

### PR DESCRIPTION
**Pull request by futa-ikeda**
## Ticket

https://openscience.atlassian.net/browse/IN-

## Notes for Reviewer

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

**Review by brianjgeiger — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,48 +1,54 @@
 # Ember OSF
 
-`master` Build Status: [![Build Status](https://travis-ci.org/CenterForOpenScience/ember-osf.svg?branch=master)](https://travis-ci.org/CenterForOpenScience/ember-osf)
+`master` Build Status: [`![Master Build Status](https://github.com/CenterForOpenScience/ember-osf/workflows/CI.yml/badge.svg?branch=master)
 [![npm Version](https://img.shields.io/npm/v/@centerforopenscience/ember-osf.svg)](https://www.npmjs.com/package/@centerforopenscience/ember-osf)
 
-`develop` Build Status: [![Build Status](https://travis-ci.org/CenterForOpenScience/ember-osf.svg?branch=develop)](https://travis-ci.org/CenterForOpenScience/ember-osf) 
+`develop` Build Status: [![Develop Build Status](https://github.com/CenterForOpenScience/ember-osf/workflows/CI.yml/badge.svg?branch=develop)
```
Comment (fabmiz) [inline README.md:6]:
```suggestion
`develop` Build Status: ![Develop Build Status](https://github.com/CenterForOpenScience/ember-osf/workflows/CI/badge.svg?branch=develop)
```

File: README.md [hunk 2]
```diff
@@ -1,48 +1,54 @@
 # Ember OSF
 
-`master` Build Status: [![Build Status](https://travis-ci.org/CenterForOpenScience/ember-osf.svg?branch=master)](https://travis-ci.org/CenterForOpenScience/ember-osf)
+`master` Build Status: [`![Master Build Status](https://github.com/CenterForOpenScience/ember-osf/workflows/CI.yml/badge.svg?branch=master)
```
Comment (fabmiz) [inline README.md:3]:
```suggestion
`master` Build Status: ![Master Build Status](https://github.com/CenterForOpenScience/ember-osf/workflows/CI/badge.svg?branch=master)
```

File: package.json [hunk 3]
```diff
@@ -16,7 +16,8 @@
     "bump-version": "npm --no-git-tag-version version ",
     "preversion": "git-branch-is develop && yarn test",
     "postversion": "yarn run docs",
-    "make-release": "git-branch-is develop && git flow release start $npm_package_version && git flow release finish $npm_package_version"
+    "make-release": "git-branch-is develop && git flow release start $npm_package_version && git flow release finish $npm_package_version",
+    "coveralls": "cat ./coverage/lcov.info | coveralls"
```
Comment (fabmiz) [inline package.json:20]: This `coveralls` can be deleted since coveralls action is taking care of uploading the coverage file to coveralls.io

**Review by fabmiz — CHANGES_REQUESTED** [review]
This looks great 💯 Minor request for typo fixes for the badges and deleting the unused coveralls script.

**Review by fabmiz — APPROVED** [review]

**Pull request merged**